examples/pktmbuf.rs

```diff
@@ -0,0 +1,11 @@
+//! An example using Mempool and Mbuf.
+
+use async_dpdk::{eal, lcore, mbuf::Mbuf};
+
+fn main() {
+    eal::Config::new().enter().unwrap();
```

Comment:
For that it's an example, please add some comment to explain what you're doing.

Reply:
Some comments are added to both examples.